tests/elasticsearch/test-class-elasticsearch.php

```diff
@@ -57,63 +59,41 @@ public function test__vip_elasticsearch_filter_ep_index_name__no_constant() {
 		$this->assertEquals( 'index-name', $index_name );
 	}
 
-	/**
-	 * Test `ep_index_name` filter for ElasticPress + VIP Elasticsearch
-	 *
-	 * USE_VIP_ELASTICSEARCH is false
-	 *
-	 * @runInSeparateProcess
-	 * @preserveGlobalState disabled
-	 */
-	public function test__vip_elasticsearch_filter_ep_index_name__constant_is_false() {
-		$mock_indexable = (object) [ 'slug' => 'slug' ];
-
-		define( 'USE_VIP_ELASTICSEARCH', false );
-
-		$index_name = apply_filters( 'ep_index_name', 'index-name', 1, $mock_indexable );
-
-		$this->assertEquals( 'index-name', $index_name );
-	}
-
 	/**
 	 * Test that we set a default bulk index chunk size limit
-	 *
+	 * 
 	 * @runInSeparateProcess
 	 * @preserveGlobalState disabled
 	 */
 	public function test__vip_elasticsearch_bulk_chunk_size_default() {
-		define( 'USE_VIP_ELASTICSEARCH', true );
-
-		vip_elasticsearch_setup_constants();
+		$es = new \Automattic\VIP\Elasticsearch\Elasticsearch();
+		$es->setup_constants();
 
 		$this->assertEquals( EP_SYNC_CHUNK_LIMIT, 250 );
 	}
 
 	/**
 	 * Test that the default bulk index chunk size limit is not applied if constant is already defined
-	 *
+	 * 
 	 * @runInSeparateProcess
 	 * @preserveGlobalState disabled
 	 */
 	public function test__vip_elasticsearch_bulk_chunk_size_already_defined() {
-		define( 'USE_VIP_ELASTICSEARCH', true );
-
 		define( 'EP_SYNC_CHUNK_LIMIT', 500 );
 
-		vip_elasticsearch_setup_constants();
+		$es = new \Automattic\VIP\Elasticsearch\Elasticsearch();
```

Comment:
Should we move this to `setUp`?

Reply:
I don't think so, because we want to `init()` or call class methods independently for each test, so having a globally scoped object that we're using in different ways in the tests could lead to cross-contamination of the state (even though technically we currently run each in a separate process).